### DISCUSSION TITLE
Suppoet Gifs, Bitmoji, And other sticker apps,

### DISCUSSION THREAD

**Discussion by StudioDribDrab** [florisboard/florisboard#1979] [Feedback]
I absolutley love floris board so customizable, so fast, and so open source, the only big thing other then the auro correct suggestions which i know your group is working hard to add is gifs and bitmoji support (along with other bitmohi alternatives) a lack of stickers and gifs really does impact the keyboard experience,

**Comment by patrickgold** [comment]
Great to hear you generally like this keyboard <3 As for the stickers, GIFs, etc: It has a few roadblocks beside lack of resources to do everything at once:

a. Implementing stickers/GIFs/Bitmoji/whatever to a privacy-friendly keyboard is a challenge due to the nasty privacy policies of these providers
b. With regards to a. I don't think emoji's will be parts of the core APK but will be a separate add-on (which when installed integrates into the core) so users who don't want to comply with the data collection of the GIF/Sticker providers just don't install the add-on
c. The core APK does not have Internet permission, so I either enable this in the core APK or in the add-on APK, this is absolutely not decided yet.

So as you can see the addition of it is more difficult than it may seem, so full support for those will still take a while.

**Reply by blackburst67** [reply]
Hi ! I come from Microsoft Keyboard (boooo I know 😅) in order to discover this wonderful keyboard.
I miss the ability to insert a gif ! This was a question in 2022, where are we in 2025 ?
Thanks!

**Reply by lm41** [reply]
https://github.com/florisboard/florisboard/issues/2485
https://github.com/florisboard/florisboard/issues/53

Hope that helps
